$OpenBSD$

Revert "Activate dispatch backend on Linux" because gcd/libdispatch has
no support for OpenBSD.

Index: Telegram/lib_crl/CMakeLists.txt
--- Telegram/lib_crl/CMakeLists.txt.orig
+++ Telegram/lib_crl/CMakeLists.txt
@@ -70,12 +70,10 @@ PUBLIC
 if (LINUX)
     target_link_libraries(lib_crl
     PUBLIC
-        desktop-app::external_dispatch
         pthread
     )
-endif()
 
-if (build_macstore OR LINUX)
+elseif (build_macstore)
     target_compile_definitions(lib_crl
     PUBLIC
         CRL_USE_COMMON_QUEUE
